recipes/miktex-portable/post-link.bat

```diff
@@ -0,0 +1,5 @@
+rem packages for nbconvert
```

Comment:
Right, so why not add miktex-portable to nbconvert's requirements and add this post-link script there?